packages/common/nbstore/src/impls/sqlite/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './blob';
+export { bindNativeDBApis, type NativeDBApis } from './db';
```

Comment:
`bindNativeDBApis` should not be exported

Reply:
bindNativeDBApis is used to implement binding API in the environment

For example, in electron
bindNativeDBApis(apis.nbstore)

In ios
bindNativeDBApis(nbstorePlugin)